samples/bot-type-ahead-search-adaptive-cards/csharp/TypeaheadSearch/Bots/ActivityBot.cs

```diff
@@ -103,28 +101,7 @@ protected override async Task<InvokeResponse> OnInvokeActivityAsync(ITurnContext
 
                     adaptiveCardResponse = new InvokeResponse()
                     {
-                        Status = 200,
-                        Body = jsonData
-                    };
-                }
-                else if(responseStatus == 500)
-                {
-                    var searchResponseData = new
-                    {
-                        type = "application/vnd.microsoft.search.searchResponse",
-                        value = new
-                        {
-                            code = 500,
-                            message = "error message: internal Server Error"
-                        }
-                    };
-
-                    var jsonString = JsonConvert.SerializeObject(searchResponseData);
-                    JObject jsonData = JObject.Parse(jsonString);
-
-                    adaptiveCardResponse = new InvokeResponse()
-                    {
-                        Status = 200,
+                        Status = 204,
                         Body = jsonData
```

Comment:
What will be in this jsondata for null case?

Reply:
we are just sending empty response of type application/vnd.microsoft.search.searchResponse